build.yaml

```diff
@@ -168,14 +196,28 @@ common_cli:
     # https://gitlab.com/gitlab-org/gitlab-ci-multi-runner/issues/20
     # So we will place the virtualenv in /tmp/v instead
     #
-    sudo rm -rf \$venv
-    sudo rm -f results.xml
-    sudo rm -f trial.log
-    sudo rm -rf _trial_temp/
+    if [ "\$(is_osx)" == "true" ]; then
+      # OSX doesn't allow us to sudo (no tty)
+      rm -f results.xml
+      rm -f trial.log
+      rm -rf _trial_temp/
+      rm -rf \${venv}
```

Comment:
the pre-scm plugin is not executed on OSX boxes as they're slaves from cloudbees